services/github/src/handlers/status.js

```diff
@@ -17,7 +17,9 @@ const { taskUI, makeDebug, taskLogUI, GithubCheck } = require('./utils');
  * This can lead to situations where newer event is being overwritten by an older one,
  * because it took him longer to reach update calls.
  */
-const qLock = new QueueLock();
+const qLock = new QueueLock({
+  maxLockTimeMs: 5 * 60 * 1000, // sometimes queue and github calls get delayed
```

Comment:
Just wanted to hear how you got to this 5 min max lock time here? It just seems sorta long, but if the calls can take up to that amount of time, then by all means we can keep it like this. I thought more of a fail-fast timing here may be better.